lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -52,6 +54,10 @@ Creates object instance of the class
 
 =item * C<timeout> - Request timeout, in seconds. If not set, uses the environment variable C<RPC_QUEUE_RESPONSE_TIMEOUT>, or defaults to 300
```

Comment:
Let's make this match the actual default while we are here (300 != 30)

Reply:
```suggestion
=item * C<timeout> - Request timeout, in seconds. If not set, uses the environment variable C<RPC_QUEUE_RESPONSE_TIMEOUT>, or defaults to 30
```